### ISSUE TITLE
Digital Annex into a JSON file: ValueError: 'ACRONYM' is not in list 

### ISSUE THREAD

**Issue by pmoriano** [nmfta-repo/pretty_1939#24]
Hello,
Thank you for open-sourcing your wonderful effort.

I wonder how to work around this error (see below). I am trying to translate a digital annex into a JSON file. Thank you.

```
$ python create_j1939db-json.py -f tmp/J1939DA_201805.xls -w tmp/J1939DA_201805.json

Traceback (most recent call last):
  File "create_j1939db-json.py", line 670, in <module>
    J1939daConverter(all_inputs).convert(args.write_json)
  File "create_j1939db-json.py", line 643, in convert
    self.process_spns_and_pgns_tab(self.find_first_sheet_by_name(sheet_name))
  File "create_j1939db-json.py", line 320, in process_spns_and_pgns_tab
    acronym_col = header_row.index('ACRONYM')
ValueError: 'ACRONYM' is not in list
```

**Comment by canon193** [comment]
I think you need to re-map the header in create_j1939db-json.py and actual header row in J1939DA_201805.xls file. Just make a debug with this code to see your current header_row value.
```
        header_row, header_row_num = self.get_header_row(sheet)
        pgn_col = header_row.index('PGN')
        spn_col = header_row.index('SPN')
        acronym_col = header_row.index('PG_ACRONYM')
        pgn_label_col = header_row.index('PG_LABEL')
        pgn_data_length_col = header_row.index('PG_DATA_LENGTH')
        transmission_rate_col = header_row.index('TRANSMISSION_RATE')
        spn_position_in_pgn_col = header_row.index('SP_POSITION_IN_PG')
        spn_name_col = header_row.index('SP_LABEL')
        offset_col = header_row.index('OFFSET')
        data_range_col = header_row.index('DATA_RANGE')
        resolution_col = header_row.index('SCALING')
        spn_length_col = header_row.index('SP_LENGTH')
        units_col = header_row.index('UNIT')
        operational_range_col = header_row.index('OPERATIONAL_RANGE')
        spn_description_col = header_row.index('SP_DESCRIPTION')
```

**Comment by BenGardiner** [comment]
Thank you @pmoriano for raising this issue and @canon193 for offering assistance. this is now finally resolved in the main branch with the merge of #26
